### PR TITLE
Add es5 bundle creation for old browsers

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
1. Return `dist.min.js` bundle creation as it was earlier to decrease bundle sizes
2. Add ES5 configuration to produce fully ES5 transpilled bundle for old browsers usage (IE 11 support as well)
# Minified Scripts OLD
modules/3d-tiles/dist/dist.min.js 419K
modules/arrow/dist/dist.min.js 290K
modules/compression/dist/dist.min.js 124K
modules/core/dist/dist.min.js 123K
modules/crypto/dist/dist.min.js 62K
modules/csv/dist/dist.min.js 61K
modules/draco/dist/dist.min.js 90K
modules/excel/dist/dist.min.js 905K
modules/flatgeobuf/dist/dist.min.js 51K
modules/gltf/dist/dist.min.js 158K
modules/i3s/dist/dist.min.js 395K
modules/images/dist/dist.min.js 88K
modules/json/dist/dist.min.js 141K
modules/kml/dist/dist.min.js 37K
modules/las/dist/dist.min.js 299K
modules/mvt/dist/dist.min.js 39K
modules/obj/dist/dist.min.js 115K
modules/pcd/dist/dist.min.js 107K
modules/ply/dist/dist.min.js 132K
modules/polyfills/dist/dist.min.js 579K
modules/potree/dist/dist.min.js 14K
modules/shapefile/dist/dist.min.js 170K
modules/tables/dist/dist.min.js 44K
modules/terrain/dist/dist.min.js 82K
modules/textures/dist/dist.min.js 131K
modules/tile-converter/dist/dist.min.js 1,4M
modules/tiles/dist/dist.min.js 273K
modules/video/dist/dist.min.js 39K
modules/wkt/dist/dist.min.js 20K
modules/zip/dist/dist.min.js 119K

# Minified Scripts NEW
modules/3d-tiles/dist/dist.min.js 130K
modules/arrow/dist/dist.min.js 240K
modules/compression/dist/dist.min.js 53K
modules/core/dist/dist.min.js 36K
modules/crypto/dist/dist.min.js 23K
modules/csv/dist/dist.min.js 19K
modules/draco/dist/dist.min.js 17K
modules/excel/dist/dist.min.js 896K
modules/flatgeobuf/dist/dist.min.js 41K
modules/gltf/dist/dist.min.js 44K
modules/i3s/dist/dist.min.js 129K
modules/images/dist/dist.min.js 12K
modules/json/dist/dist.min.js 23K
modules/kml/dist/dist.min.js 19K
modules/las/dist/dist.min.js 231K
modules/mvt/dist/dist.min.js 25K
modules/obj/dist/dist.min.js 14K
modules/pcd/dist/dist.min.js 9,2K
modules/ply/dist/dist.min.js 8,5K
modules/polyfills/dist/dist.min.js 569K
modules/potree/dist/dist.min.js 3,1K
modules/shapefile/dist/dist.min.js 105K
modules/tables/dist/dist.min.js 17K
modules/terrain/dist/dist.min.js 9,6K
modules/textures/dist/dist.min.js 35K
modules/tile-converter/dist/dist.min.js 845K
modules/tiles/dist/dist.min.js 119K
modules/video/dist/dist.min.js 28K
modules/wkt/dist/dist.min.js 8,2K
modules/zip/dist/dist.min.js 106K

### CODE REVIEW

File: modules/3d-tiles/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "README.md"
   ],
   "scripts": {
-    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev",
+    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev && npm run build-bundle -- --env.es5",
```
Comment (ibgreen) [inline modules/3d-tiles/package.json:32]: We should investigate updating the webpack script to return an array of configurations so we don't need to update the script in all places.

File: scripts/webpack/bundle.js [hunk 2]
```diff
@@ -110,6 +108,17 @@ module.exports = (env = {}) => {
...
+    config.output.filename = 'dist/dist.es5.min.js';
+    config.module.rules = [
+      {
+         // Compile ES2015 using babel
+         test: /\.js$/,
+         loader: 'babel-loader',
+         include: /src/,
+         options: ES5_BABEL_CONFIG
+      }
+    ];
```
Comment (ibgreen) [inline scripts/webpack/bundle.js:121]: We should be able to return an array of configurations at the bottom here.

**Review by ibgreen — APPROVED** [review]

File: modules/3d-tiles/package.json (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/3d-tiles/package.json]: Done

File: scripts/webpack/bundle.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply scripts/webpack/bundle.js]: DOne

**Review by ibgreen — APPROVED** [review]
Great cleanup, thanks for going the extra mile!

**Comment by dryabinin-actionengine** [comment]
@ibgreen What do you think about merging this changes to 2.3-release branch and notify people who has es5 transpillation issues?

**Pull request merged**